.travis.yml

```diff
@@ -12,6 +12,7 @@ before_script:
   - sleep 3
 script:
   - cd vms
+  - flake8 
```

Comment:
Did you test this for your fork?

Reply:
Yes, but it is also failing. You can check it at: https://travis-ci.org/kamsuri/vms/jobs/290791344.

Reply:
instead here say ```flake8 .```

Reply:
instead here say flake8 .